src/backend/package.json

```diff
@@ -21,9 +21,11 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@nestjs/axios": "0.1.0",
```

Comment:
can we stick to pure javascript async and await. Axios is not really adding anything much to async/await.

Also there are multiple nested sub dependencies which will cause trouble during npm updates in the future.

Reply:
i will check this